### ISSUE TITLE
Accessibility: Upgrade to the latest a11y policy

### ISSUE THREAD

**Issue by jlpadilla** [carbon-design-system/carbon-components-react#87]
The a11y tests are using an old version of the policy rules. See code [here](https://github.com/carbon-design-system/carbon-components-react/blob/master/.aat.yml#L10).
```
policies:
  - CI162_5_2_DCP070116
  - CI162_5_2_DCP080115
```

The latest policy rules from Feb 2017 is: `IBM_Accessibility_2017_02`.

When using the latest a11y policy in our project that depends `carbon-components-react` we see several violations coming from components in this library.

**Comment by shyfruan** [comment]
Our team also met the accessible problem. @chrisdhanaraj

**Comment by jlpadilla** [comment]
Some of the most common violations that I'm seeing are:

- A widget with image role must have alternate text that describes the image. [doc here](https://aat.w3ibm.mybluemix.net/token/91db6268-7b88-45b7-b2c8-f0df4255d82c/093092e6-8622-42ef-9e0a-6690bec8ea57/doc/w3/help/en-US/idhi_accessibility_check_g1128.html)
  - This appears for every icon, even if it has an associated `<title>`.

- Text content does not meet WCAG AA minimum contrast requirements [doc here](https://aat.w3ibm.mybluemix.net/token/91db6268-7b88-45b7-b2c8-f0df4255d82c/093092e6-8622-42ef-9e0a-6690bec8ea57/doc/w3/help/en-US/idhi_accessibility_check_g1128.html)
  - I see this one in various places, but the InteriorLeftNav probably has the most violations.

- Label 'For' attribute does not reference an appropriate form control [doc here](https://aat.w3ibm.mybluemix.net/token/91db6268-7b88-45b7-b2c8-f0df4255d82c/093092e6-8622-42ef-9e0a-6690bec8ea57/doc/w3/help/en-US/idhi_accessibility_check_g1062.html)
  - This one is on the pagination component.

**Comment by hellobrian** [comment]
@jlpadilla we've switched from IBM a11y tools to using aXe
- [storybook-addon-a11y (wrapper for axe-core)](https://github.com/storybooks/storybook-addon-a11y) 

We found that this has been thorough to match with component-level violations we know we should be watching out for: 
- WCAG AA color contrast ratio compliance
- Semantic HTML
- Adding the correct HTML attributes for AT
